### ISSUE TITLE
Birthday format needs to be consistent

### ISSUE THREAD

**Issue by adamreisnz** [passport-next/passport-facebook#8]
The birthday field in the profile is populated straight form what comes from Facebook.
In this case it's formatted mm/dd/yyyy.

I propose we re-format this to ISO standard format, e.g. YYYY-MM-DD

Can use the following code to achieve this transformation:

```js
if (json.birthday) {
  const parts = json.birthday.split('/');
  profile.birthday = [parts[2], parts[0], parts[1]].join('-');
}
```

**Comment by rwky** [comment]
Agreed, YYYY-MM-DD is far less ambiguous than MM-DD-YYYY (which where I come from would be interpreted as DD-MM-YYYY).

**Comment by adamreisnz** [comment]
Yes same here, great I'll go ahead and implement this, but as it could be a breaking change for some it'll likely need a major version bump. So might want to hold off until it's clear what's happening in terms of unification of the profiles?

**Comment by rwky** [comment]
Agreed. I'll hopefully get time next week to have a good think on things. I also need to lint all the repos before we go making too many changes.

**Comment by adamreisnz** [comment]
@rwky do you still maintain this package?

**Comment by rwky** [comment]
Yep, not worked on unifying profiles lately, promise support has been a higher priority though if you want to add a PR I'll take a look :)

**Comment by mddmayln** [comment]
I suppose either way 
Considering same form to understand
